gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -107,6 +231,25 @@ def main(args):
         )
         logger.info("CADD Hail Table for GRCh38 created.")
 
+    if args.spliceai:
+        logger.info("Creating SpliceAI Hail Table for GRCh38...")
+        snvs_path = (
+            "gs://gnomad-qin/v4_annotations/spliceai_scores.masked.snv.hg38.vcf.bgz"
+        )
+        indels_path = (
+            "gs://gnomad-qin/v4_annotations/spliceai_scores.masked.indel.hg38.vcf.bgz"
+        )
+        new_indels_path = "gs://gnomad-qin/v4_annotations/gnomad_v4_new_indels.spliceai_masked.vcf.bgz"
+        header_file_path = "gs://gnomad-qin/v4_annotations/spliceai.vcf.header"
```

Comment:
I think we should keep this full resource spliceAI resource internal given the discussion around licensing earlier this year (copy the current files into a new bucket with autoclass enabled).

for the other resources, we shouldn't need to copy the import files, so something like
```
cadd = VersionedTableResource(
    versions={
        "snvs": GnomadPublicTableResource(
            path="gs://gcp-public-data--gnomad/resources/grch38/in_silico_predictors/CADD_v1.6_SNVs.ht",
        ),
        "indels3.0": GnomadPublicTableResource(
            path="gs://gnomad-public-requester-pays/resources/grch38/cadd/cadd-indels-gnomad.3.0.ht",
            import_func=_load_cadd_raw,
            import_args={
                "path":""gs://gnomad-qin/v4_annotations/cadd.v1.6.gnomad.genomes.r3.0.indel.tsv.bgz"
            }
        ),
    }
)
```

Reply:
So I agree with KC above except for the import_args path.  I don't think any files used in production should live in personal buckets. When people clean their personal buckets up is unpredictable and I don't think we should have to dig into the codebase to find a source file. Having the file in an organized and well labeled bucket is nice. It shouldnt be public but it can live in  `gs://gnomad/v4.0/annotations/in_silico_predictors` which is an intuitive, existing bucket.

Reply:
We created a new bucket for the nonpublic raw data: gs://gnomad-insilico, I'm moving them there and adding to gnomad/resources/grch38/reference_data.py, I will create another PR in gnomad_methods.

Reply:
Aabout copying resources to public bucket, for dbSNP, I just did as before, https://github.com/broadinstitute/gnomad_methods/blob/12a11ae74d09ed46e92a2e3eb49182dc05aaa4b8/gnomad/resources/grch38/reference_data.py#L223, but it doesn’t make sense to me to keep both raw data & HT in the public bucket, for the other insilico predictors, I’m going to import from gnomad-insilico instead of from gcp-public-data--gnomad.

Reply:
I think I know why now, it has to be imported from the public buckets: 
`Job failed with message [ValueError: GnomadPublicResource requires a path to a file in one of the public gnomAD buckets (gnomad-public, gnomad-public-requester-pays)]. Additional details can be found at:`